docs/core/deploying/trimming/trimming-options.md

```diff
@@ -79,10 +119,14 @@ Do not add or remove items to/from `ManagedAssemblyToLink`, because the SDK comp
 
   Control whether the given assembly is trimmed.
 
+:::zone pivot="dotnet-6-0,dotnet-5-0"
+
 - `<TrimMode>copyused</TrimMode>` or `<TrimMode>link</TrimMode>`
 
   Control the [trimming granularity](#trimming-granularity) of this assembly. This takes precedence over the global `TrimMode`. Setting `TrimMode` on an assembly implies `<IsTrimmable>true</IsTrimmable>`.
 
+:::zone-end
+
```

Comment:
Can't comment on the line +7 lines below this one but "All assemblies that do not have `<IsTrimmable>true</IsTrimmable>` are considered roots for the analysis" needs changing too.